### ISSUE TITLE
Livepremiere - "Take" Issue

### ISSUE THREAD

**Issue by seriusbow** [bitfocus/companion-module-analogway-awj#30]
Hi,
in some cases the command "Take" with Livepremiere is not perfomed correctly in every screen. I've found this problem in the simulator and also with the real hardware.
During a Take of multiple screens, if one screen is performing a "TakeDown", all the subsequent screens will always perform a "TakeDown" and so if a specific screen needs a "TakeUp", the screen doesn't perform the transition.

Companion version: 3.4.3 
Aquilon version: 4.2.17 and 4.3.38

Steps to reproduce the issue:
- Open the Livepremiere simulator and create 2 screens. 
- Using the WebRCS, move the T-bar to the UP position with both screens selected. The transition will be performed correctly for screen 1 and 2.
- Now, using the WebRCS, move the T-bar to the DOWN position with both screens selected. The transition will be performed correctly  for screen 1 and 2.
- Use the command "Take" selecting only the first screen in Companion. The transition will be performed correctly for screen 1.
- Use the command "Take" selecting screens 1 and 2 in Companion. The transition will be be performed for screen 1, but not for screen 2.

Companion sends TakeDown for both screens, but screen 2 needs TakeUp to perform the transition.

**Comment by dnmeid** [comment]
Thanks for the report. Did you really verify that the wrong take direction is sent or is that just an assumption?

**Comment by seriusbow** [comment]
Yes, I've verified.
With Wireshark software I've capture the packets sent from companion to the Livepremiere Simulator and checked the commands inside the packets.
Moreover, usign Hercules software, I've subscribe to machine state change notification using the AWJ protocol on port 10606.

When I press the "Take Screen 1 and 2"  button on companion on step 5 of my previous message I have captured two packets with destination port 10506 (that is bind to AW Simulator) with the command "TakeDown" for screen 1 and 2.
And in Hercules software I see a state change notification of "TakeDown" for both screens.
